ethbridge/src/lib.rs

```diff
@@ -190,10 +194,10 @@ impl EthBridge {
 
     pub fn hashimoto_merkle(
         &self,
-        header_hash: H256,
-        nonce: H64,
+        header_hash: &H256,
+        nonce: &H64,
         block_number: u64,
-        nodes: Vec<DoubleNodeWithMerkleProof>,
```

Comment:
I see potential problem to switch back to `near-bindgen`, when arguments parsing will use Borsh  by default

Reply:
Yes, let's not switch to auto-borsh parsing until after EthDenver.

Reply:
I split `impl EthBridge {` into 2 sections. One to use with `near-bindgen` macro and one to use without. Basically, they are private methods that only used internally and don't expose function API publicly from contract.